source/common/network/address_impl.h

```diff
@@ -35,15 +36,15 @@ Address::InstanceConstSharedPtr addressFromSockAddr(const sockaddr_storage& ss,
 
 /**
  * Obtain an address from a bound file descriptor. Raises an EnvoyException on failure.
- * @param fd file descriptor.
```

Comment:
mind reinstating the punctuation here and below?

Reply:
That's odd- I actually haven't removed that line- I just compared that part of address_impl.h to master and the code is identical. Here's the snippet of that file in my branch:

/**
 * Obtain an address from a bound file descriptor. Raises an EnvoyException on failure.
 * @param fd socket file descriptor
 * @return InstanceConstSharedPtr for bound address.
 */
InstanceConstSharedPtr addressFromFd(int fd);

Reply:
Formatted properly, of course.

Reply:
Can be addressed in a follow up.